src/styles/aui/_aui.navigation-menu.scss

```diff
@@ -0,0 +1,427 @@
+@use '../quarks/quarks.variables';
+
+$mobile-menu-height: 60px;
+$navigation-pane-width: 20rem;
+$menu-undocked-width: 15rem;
+$menu-docked-width: $header-height;
+$submenu-inset: 12px;
+$submenu-height: 30rem;
+$animation-speed: 150ms;
+$overlay-color: #444;
```

Comment:
Enkele zaken hier:
- kan je de pixel eenheden allemaal in `rem` zetten en bij voorkeur er zelfs veelvouden van `$spacer van maken`?
- kan je naar analogie met de andere componenten de belangrijkste variables mee in de `_quarks.variables.scss` zetten en deze allemaal hetzelfde prefix geven.
- Graag de `$overlay-color` gebruiken die al in voorgenoemde file staat.